Fatras/include/ActsFatras/Kernel/ContinuousProcess.hpp

```diff
@@ -65,7 +76,7 @@ struct Process {
     // modify particle according to the physics process
     auto children = physics(generator, slab, particle);
     // move selected child particles to the output container
-    std::copy_if(children.begin(), children.end(),
+    std::copy_if(std::begin(children), std::end(children),
```

Comment:
What's the motivation for this? Is it needed to allow generic types for `children`?

Reply:
Yes, this change is there to allow any iterable type to work with this. This is not really something that is needed at the moment, but I thought it would be better to do the right thing here. Btw., this is also how [range-based for loops](https://en.cppreference.com/w/cpp/language/range-for) are defined.